docs/developers.md

```diff
@@ -7,6 +7,7 @@ Helm and Tiller.
 
 - Go 1.6.0 or later
 - Glide 0.12.0 or later
+- Gometalinter
```

Comment:
While this might not seem like much, what about updating the docs in a separate PR? One thing per change and this makes it two.

Reply:
Removed.